BAZProjectC1/Components/CarouselMovies/movieCarouselCell.swift

```diff
@@ -0,0 +1,100 @@
+//
+//  movieCarouselCell.swift
+//  BAZProjectC1
+//
+//  Created by 1044336 on 22/09/22.
+//
+
+import UIKit
+
+class movieCarouselCell: UICollectionViewCell {
```

Comment:
La convención de nombres de clases el UpperCammelCase. Considera renombrar esta clase:

```suggestion
class MovieCarouselCell: UICollectionViewCell {
```